src/include/sof/audio/volume.h

```diff
@@ -60,8 +59,8 @@ struct sof_ipc_ctrl_value_chan;
  * VOL_RAMP_UPDATE / 1000.0 for step calculation. The value 1000
  * is used to to convert microseconds to milliseconds.
  */
-#define VOL_RAMP_STEP_CONST \
-	Q_CONVERT_FLOAT(VOL_RAMP_UPDATE_US / 1000.0, VOL_QXY_Y)
+#define VOL_RAMP_STEP_CONST(x) \
+	Q_CONVERT_FLOAT((x) / 1000.0, VOL_QXY_Y)
```

Comment:
I'm worried about this since the pre-processor can't compute this. Does this emit emulated float code?